5-infrastructure/modules/base-env/boa-gke.tf

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  gke_settings = {
+    cluster_1 = {
+      name                   = "gke-boa-${var.location_primary}-001"
+      subnetwork             = var.gke_cluster_1_subnet_name
+      ip_range_pods          = var.gke_cluster_1_range_name_pods
+      ip_range_services      = var.gke_cluster_1_range_name_services
+      master_ipv4_cidr_block = var.gke_cluster_1_cidr_block
+      region                 = var.location_primary
+      machine_type           = "e2-standard-4"
+      master_authorized_networks = [
+        {
+          cidr_block   = element([for subnet_ip_range in flatten([for subnet in data.google_compute_subnetwork.subnet : subnet.secondary_ip_range if length(subnet.secondary_ip_range) > 0 && subnet.name == var.gke_cluster_2_subnet_name]) : subnet_ip_range.ip_cidr_range if subnet_ip_range.range_name == var.gke_cluster_2_range_name_pods], 0),
+          display_name = var.gke_cluster_2_range_name_pods
```

Comment:
As discussed offline, lets give a more verbose name `cluster 1 pods to cluster 2 controlplane` etc